src/js/pages/Arbitration/index.jsx

```diff
@@ -36,20 +40,58 @@ class Arbitration extends Component {
     props.loadArbitration(props.escrowId);
   }
 
+  state = {
+    displayUsers: false,
+    selectedUser: null,
+    displayDialog: false
+  }
+
   componentDidMount() {
   }
 
+  openSolveDisputeDialog = () => {
+    this.setState({displayUsers: true});
```

Comment:
Shouldn't this be `displayDialog`? Or it's just the name of the function that's confusing?

Reply:
there are two display functions here. One to select the user that will win the dispute, and another to confirm the selection.